src/main/java/uk/co/ramp/covid/simulation/parameters/PopulationParameters.java

```diff
@@ -41,7 +41,7 @@ public boolean isValid() {
         boolean valid = true;
         // We don't do this in a single statement to ensure that all the "uninitalised" parameter warnings are printed
         // in one go instead of being short circuited
-        valid = valid && checker.isValid(populationDistribution);
+        valid = checker.isValid(populationDistribution);
```

Comment:
If you're changing this, then it doesn't make sense anymore to have line 41 setting `valid = true`.  (I'm a little surprised that IntelliJ didn't complain about this!)
```suggestion
        boolean valid = checker.isValid(populationDistribution);
```